client/src/components/scans/scanJobsList.js

```diff
@@ -106,7 +106,7 @@ class ScanJobsList extends React.Component {
           {job.systems_scanned > 0 ? job.systems_scanned : '0'}
         </Grid.Col>
         <Grid.Col xs={3} sm={2} smPush={5}>
-          {job.status === 'completed' && (
+          {job.report_id > 0 && (
```

Comment:
It probably shouldn't happen a zero ID, buttttt can totally see it happening... maybe just a `>=`. 

This check along with the one on `line 209, client/src/components/scans/scanListItem.js` seem like they should be doing the same set of checks if we're just detecting the property exists, or that it has a value to "report" off of.

Reply:
0 is not a valid id, but I can change to >= 0 or a comparison against undefined if you prefer.

Reply:
Expanding out, looks like it's in a couple of spots, can avoid it